packages/components/src/SearchParamsSummary/SearchParamsSummary.js

```diff
@@ -41,40 +41,38 @@ export default function SearchParamsSummary({
 
   return (
     <View style={styles.container}>
-      <View style={styles.headerLeftcontainer}>
+      <View style={styles.headerLeftContainer}>
         <View style={styles.citiesContainer}>
           <Text style={styles.departureCity}>{departure?.city || ''}</Text>
           <Icon name={icon} />
           <Text style={styles.arrivalCity}>{arrival?.city || ''}</Text>
         </View>
-        {arrival != null && departure != null && (
-          <View>
-            {tripType === 'OneWay' ? (
-              <View>
-                <AdaptableBadge
-                  style={styles.badge}
-                  textStyle={styles.badgeText}
-                  text={arrival.localizedDate}
-                />
-              </View>
-            ) : (
-              <View style={styles.row}>
-                <AdaptableBadge
-                  style={styles.badge}
-                  textStyle={styles.badgeText}
-                  text={departure.localizedDate}
-                />
-                <Text style={styles.connector}> to </Text>
-                {/* @TODO localize string `to` */}
-                <AdaptableBadge
-                  style={styles.badge}
-                  textStyle={styles.badgeText}
-                  text={arrival.localizedDate}
-                />
-              </View>
-            )}
-          </View>
-        )}
+        {arrival &&
+          departure &&
+          (tripType === 'OneWay' ? (
+            <View>
+              <AdaptableBadge
+                style={styles.badge}
+                textStyle={styles.badgeText}
+                text={departure.localizedDate}
```

Comment:
This won't be displayed if there is no `arrival` though it should be possible, right?

Reply:
You are right. Done